python/ray/rllib/models/catalog.py

```diff
@@ -32,6 +32,8 @@ class ModelCatalog(object):
         action_op = dist.sample()
     """
 
+    _registered_preprocessor = dict()
```

Comment:
This is basically a global variable, right? What's the advantage of putting it inside of `ModelCatalog` instead of as an actual global variable?

Reply:
This is a class variable and cannot be accessed outside of the class; ModelCatalog is a singleton class here.

Reply:
Got it.